src/App.tsx

```diff
@@ -15,36 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortBy {
+  Alphabetically = 'alphabetically',
+  Length = 'length',
+}
+
+interface SortParams {
+  sortField: string;
+  isReversed: boolean;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: SortParams,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortBy.Alphabetically:
+          return good1.localeCompare(good2);
+
+        case SortBy.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    return preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+
+  const sortGoods = getPreparedGoods(goodsFromServer, {
```

Comment:
```suggestion
  const sortedGoods = getPreparedGoods(goodsFromServer, {
```
`sortGoods` looks like a function name, as it starts from a verb